### DISCUSSION TITLE
Sidebar like in the TAB plugin

### DISCUSSION THREAD

**Discussion by saxonov80** [Zrips/CMI#8373] [Feature Requests]
Is it possible to add a sidebar in CMI, like in the TAB plugin. I would like a name for the animation, support for placeholders, and the ability to turn the panel on or off.
This would allow us to remove the unnecessary plugin from the server. Thank you.

**Comment by CloudeLecaw** [comment]
https://github.com/Zrips/CMI/discussions/7185

https://github.com/Zrips/CMI/discussions/7058

sometimes it would help us more to discussion an existing discussion even if its x years old.
just ender a keyword in the discussion line and take a quick look if something simliar may allready exist.
